sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -397,6 +397,13 @@ RUN sudo augtool --autosave "set /files/etc/dpkg/dpkg.cfg/force-confold"
 RUN apt-get -y build-dep linux
 
 # For gobgp and telemetry build
+{%- if INCLUDE_FIPS == "y" %}
+RUN wget -O golang-go.deb 'https://sonicstorage.blob.core.windows.net/public/fips/bullseye/0.1/golang-1.15-go_1.15.15-1~deb11u2%2Bfips_{{ CONFIGURED_ARCH }}.deb' \
```

Comment:
>1.15 [](http://example.com/codeflow?start=102&length=4)

Can we use the same version as in `else` branch? Ideally, we should share the version constant, and build any version specified with FIPS.

Reply:
Change all to use 1.15 for bullseye, align to the default one in bullseye.
And change to use the debian packages, not to download from google web site.